config.py

```diff
@@ -1250,6 +1270,19 @@
     ```
 """
 
+n_freqs: int = 5
+"""
+The number of frequencies used for decoding using time frequency windows.
+Example: n_freqs = 5 generates 4 frequency bins.
```

Comment:
I think I, as a user, would prefer to specify the number of frequency bins?

Reply:
yes this was a definitely a bad api... I've deleted that part this morning